graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationEventProcessorConfig.java

```diff
@@ -136,4 +138,18 @@ public static Builder create() {
     public ValidationResult validate() {
         return new ValidationResult();
     }
+
+    @Override
+    public Object toContentPackEntity() {
+        return AggregationEventProcessorConfigEntity.builder()
+            .type(type())
+            .query(ValueReference.of(query()))
+            .streams(streams()) // TODO?
```

Comment:
Is this `TODO` still relevant?

Reply:
Yes, but I wrote it down with pen and paper :)